Persephone/Views/AlbumItem.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AlbumItem.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/08.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+
+class AlbumItem: NSCollectionViewItem {
+  let borderColorLight = NSColor.black.withAlphaComponent(0.1).cgColor
+  let borderColorDark = NSColor.white.withAlphaComponent(0.1).cgColor
```

Comment:
What do you think about moving all app colors into a centralized extension on UIColor?

Reply:
Moved to an extension on `CGColor`.